renters_rights/units/contactus.py

```diff
@@ -0,0 +1,9 @@
+rom django import forms
+
+class NameForm(forms.form):
+    your_email = forms.CharField(label = 'Your Email', max_length = 100)
```

Comment:
One small change to your labels. We want these to be able to be translated into other languages easily, and Django facilitates that through their translation module.

In the `forms` module, you'll see a line like `from django.utils.translation import gettext_lazy as _`. And then strings wrapped in `_()`. This makes them easy to export for translation and translatable at runtime.

So for your labels, could you wrap them in `_()`? For example, `label='Your Name` would become `_('Your name')`.

When you move this form class to the `forms` module, `_` will have been imported for you, so you won't need to import it.